src/cljs/ataru/hakija/components/dropdown_component.cljs

```diff
@@ -26,7 +26,8 @@
                        first
                        :followups
                        (filter #(deref (re-frame/subscribe [:application/visible? (keyword (:id %))]))))
-        data-test-id (when (some #{id} [:home-town])
+        data-test-id (when (some #{id} [:home-town
+                                        :language])
                        (-> id
                            name
                            (str "-input")))]
```

Comment:
Kannattaisikohan tässä kohtaa laittaa data-test-id :hen vielä jotakin löytämistä helpottavaa tekstiä?

Kun nimittäin näin testikoodissa käytetyn data-test-id :tä `language-input`, sen löytäminen greppailemalla ei ollut ihan triviaalia.

Sama pätee tietty yleisimminkin näihin dynaamisiin data-test-id :ihin.